### PR TITLE
Cd/release env

### PR DESCRIPTION
**Pull request by corpulent**
Environment deployment pipeline.

https://jira.infor.com/browse/HLO-302

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,73 +1,122 @@
-# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
-# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
-
-name: Build Check
+name: Deploy environment
 
 on:
   push:
-    branches: [ main ]
+    branches:
```
Comment (tmcconechy) [inline .github/workflows/ci.yml:5]: Is this right? Seems like that means any branches we push will deploy?

**Review by tmcconechy — DISMISSED** [review]
Would like to discuss the issue if I read it right every single one here would deploy https://github.com/infor-design/enterprise/branches and no one would need any of them

Maybe we only need to do this on main or via some branch pattern? Either that or need to communicate NOT to push your branch for no reason. I dont think i see the need for every branch... But willing to discuss

File: .github/workflows/ci.yml (hunk 1 above)
Comment (corpulent) [reply .github/workflows/ci.yml:5]: Yes this is correct.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -1,73 +1,121 @@
...
```
Comment (tmcconechy) [inline .github/workflows/ci.yml:5]: Are we going to change this then? Not thinking its right too just release everything we push

**Review by danielortiz1982 — DISMISSED** [review]

**Comment by tmcconechy** [comment]
The only problem i have with this PR now are some of the rules on the package.json. It is common that you will need a few other changes to support the changes in the package.json and this check doesn't seem to allow that.

For example https://github.com/infor-design/enterprise/pull/6731 would not be possible, some things change in package json that need other changes. Because of that this seems too restrictive. Maybe we can do this as a pr review note/manual check?

**Review by danielortiz1982 — APPROVED** [review]

**Review by tmcconechy — APPROVED** [review]

**Pull request merged**